test/unit/server_test.js

```diff
@@ -1507,6 +1509,149 @@ describe('server.js non-transaction tests', function() {
           });
       });
 
+      it('adds an "liquidity_pool" query to the endpoint', function(done) {
```

Comment:
nit: description typo
```suggestion
      it('adds a "liquidity_pool" filter to the endpoint', function(done) {
```

Reply:
Addressed in https://github.com/stellar/js-stellar-sdk/pull/688/commits/8ec0cdc93f295652cade3d12f135d4dec0695752.